api/server/server.go

```diff
@@ -554,7 +554,7 @@ func (s *Server) getContainersStats(version version.Version, w http.ResponseWrit
 		return fmt.Errorf("Missing parameter")
 	}
 
-	return s.daemon.ContainerStats(vars["name"], boolValue(r, "stream"), ioutils.NewWriteFlusher(w))
+	return s.daemon.ContainerStats(vars["name"], boolValueOrDefault(r, "stream", true), ioutils.NewWriteFlusher(w))
```

Comment:
in 1.6 there wasn't this query param at all and *streaming* was always true
so now the behavior to maintain bc in 1.7 is we need to allow `GET /container/id/stat` to stream by default (this PR) and make  `GET /container/id/stat?stream=""` false of course (hence boolValueOrDefault)

@cpuguy83 am I missing something?